instruqt-tracks/terraform-cloud-aws-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -19,14 +36,85 @@ cd /root/hashicat-aws
 sed -i "s/YOUR_ORGANIZATION/$ORG/g" remote_backend.tf
 sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" remote_backend.tf
 
-cd /root/terraform-api
-sed -i "s/YOUR_ORGANIZATION/$ORG/g" terraform.tfvars.example
-mv terraform.tfvars.example terraform.tfvars
+cp /root/hashicat-aws/terraform.tfvars.example /root/hashicat-aws/terraform.tfvars
+
+# This was originally going into the terraform-api directory... might need to check on this in the API challenge
+cd /root/terraform-cloud
+sed -i "s/YOUR_ORGANIZATION/$ORG/g" terraform.tfvars # This was originally looking in terraform.tfvars.example
+sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" terraform.tfvars
+# mv terraform.tfvars.example terraform.tfvars
+# The above command will get rid of the terraform.tfvars.example file, but when
+# we run the check script more than once, it cannot find that file bc it no longer
+# exists. However, to keep the consistency between code blocks, we kept this the 
+# same as above where we search and replace in the remote_backend.tf file. 
 
 # Store the ORG in /root/.bashrc
```

Comment:
For reference in later comments, lines 51-55 here are only still in here because we haven't finished VCDL-125 and done the proper thing in setting these environment variables in the root `.bashrc`

Reply:
This comment isn't a blocker, however.